x/profiles/simulation/params.go

```diff
@@ -18,29 +19,33 @@ func ParamChanges(r *rand.Rand) []simtypes.ParamChange {
 		simulation.NewSimParamChange(types.ModuleName, string(types.NicknameParamsKey),
 			func(r *rand.Rand) string {
 				params := RandomNicknameParams(r)
-				return fmt.Sprintf(`{"min_nickname_len":"%s","max_nickname_len":"%s"}`,
+				return fmt.Sprintf(`{"min_length":"%s","max_length":"%s"}`,
```

Comment:
are we able to distinguish the params on tests if we have more or less the same names for both nickname and dtag params?

Reply:
@bragaz Yes, because they are associated to a specific key. In this case at line 19 we have `string(types.NicknameParamsKey)`